src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATE_OF_PAY = 10;
+    private static final int HOURS_WORKED = 2;
+    private static final int SALARY_PER_HOUR = 3;
+    private static final int START_OF_RANGE = 0;
+    private static final String SPACE = " ";
+    private static final String REPORT_MESSAGE = "Report for period ";
+    private static final String SEPARATE = " - ";
+    private StringBuilder sb = new StringBuilder();
+    private int salary = 0;
```

Comment:
do we need this class fields?
it would be better to move them to `getSalaryInfo` method